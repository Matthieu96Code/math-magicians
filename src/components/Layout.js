import { Outlet } from 'react-router-dom';
import Navbar from './Navbar';
import '../styles/Math.css';

const Layout = () => (
  <div className="layout">
    <div className="mainbar">
      <h1 className="mainTitle">Math Magicians</h1>
      <Navbar />
    </div>
    <Outlet />
  </div>
);

export default Layout;
