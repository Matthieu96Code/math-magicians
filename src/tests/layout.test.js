import { BrowserRouter as Router } from 'react-router-dom';
import React from 'react';
import { render } from '@testing-library/react';
import Layout from '../components/Layout';

test('Layout renders correctly', () => {
  const { container } = render(
    <Router>
      <Layout />
    </Router>,
  );
  expect(container.firstChild).toMatchSnapshot();
});
