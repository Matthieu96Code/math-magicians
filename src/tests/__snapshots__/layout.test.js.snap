// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  class="layout"
>
  <div
    class="mainbar"
  >
    <h1
      class="mainTitle"
    >
      Math Magicians
    </h1>
    <nav
      class="navbar"
    >
      <ul
        class="menuList"
      >
        <li
          class="listElement"
        >
          <a
            class="linkedList"
            href="/home"
          >
            Home
          </a>
        </li>
        <li
          class="listElement"
        >
          <a
            class="linkedList"
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li
          class="listElement"
        >
          <a
            class="linkedList"
            href="/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
